source/2020-06-12-the-ember-times-issue-152.md

```diff
@@ -47,14 +48,13 @@ If you want to find out how to level up your app's accessibility, read [her blog
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Introducing ember-concurrency-async 🚀](https://github.com/chancancode/ember-concurrency-async)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Have you wanted to use [`ember-concurrency`](https://github.com/machty/ember-concurrency) with TypeScript, but run into some limitations? In case you were not aware, there's a new addon by [Godfrey Chan (@chancancode)](https://github.com/chancancode) called `ember-concurrency-async`. This addon introduces the familiar `async/await` syntax into `ember-concurrency` tasks. 😃
```

Comment:
```suggestion
Have you wanted to use [`ember-concurrency`](https://github.com/machty/ember-concurrency) with TypeScript, but run into some limitations? In case you were not aware, there's a new addon by [Godfrey Chan (@chancancode)](https://github.com/chancancode) called [`ember-concurrency-async`](https://github.com/chancancode/ember-concurrency-async). This addon introduces the familiar `async/await` syntax to `ember-concurrency` tasks. 😃
```